prow/jobs.md

```diff
@@ -93,11 +94,119 @@ command that reruns all jobs. If unspecified, the default configuration makes
 
 See the [Triggering Jobs](#triggering-jobs) section below to learn how to
 control when jobs are automatically run. We also have sections about [posting](#posting-github-status-contexts)
-and [requiring](#requiring-job-statuses) GitHub status contexts. A useful
-pattern when adding new jobs is to start with `always_run` set to false and
-`skip_report` set to true. Test it out a few times by manually triggering,
-then switch `always_run` to true. Watch for a couple days, then switch
-`skip_report` to false.
+and [requiring](#requiring-job-statuses) GitHub status contexts.
+
+A useful pattern when adding new jobs is to start with `always_run` set to
+`false` and `skip_report` set to `true`. Test it out a few times by manually
+triggering; this manual triggering phase allows you to iron out any bugs without
+getting flooded with the same failures (see "Triggering Jobs With Comments"
+below). Then when things look good, switch `always_run` to `true`. Watch for a
+couple days, then switch `skip_report` to `false`.
+
+#### Postsubmits
+
+Here is a sample **postsubmit** config (see [GoDocs](https://pkg.go.dev/k8s.io/test-infra/prow/config#Postsubmit) for complete config):
+
+```yaml
+postsubmits:
+  org/repo:
+  - name: bar-job         # As for periodics.
+    decorate: true        # As for periodics.
+    spec: {}              # As for periodics.
+    max_concurrency: 10   # Run no more than this number concurrently.
+    branches:             # Regexps, only run against these branches.
+    - ^main$
+    skip_branches:        # Regexps, do not run against these branches.
+    - ^release-.*$
+```
+
+Postsubmits are configured per-repo, because most of the time they are
+configured to run when a branch updates on a repo (due to a push or PR merge).
+If no `branches` are specified, then they will run against every branch.
+
+Postsubmits do not always have to depend on branch updates. The trick is to set
```

Comment:
Its confusing that we are only mentioning this specifically for Postsubmits. We should move this or clarify that this is true of Presubmits as well.

Reply:
This is all pretty specific to an uncommonly used pattern so it might be better to just briefly mention this feature here and then link to the full explanation in the Sub docs to avoid overwhelming users with a bunch of details that aren't applicable.